### PR TITLE
Added version to Facebook auth

### PR DESCRIPTION
**Pull request by weswingcats**
Removes deprecation warning

### CODE REVIEW

**Comment by simov** [comment]
Hi @weswingcats where exactly do you see that deprecation warning?

**Comment by weswingcats** [comment]
Hi!

It was on the initial permissions screen (below). 

I rolled back the changes on my local and cannot replicate it again. It stated, that "this version of the API will be deprecated on 30 April"

If you feel confident to ommit it, then no problem. :+1: 

screen shot 2015-04-13 at 10 46 57

**Comment by simov** [comment]
That's the authorization screen, but you are changing the access_token url which is called internally after this screen. So there is no way for you to see a warning from it, unless it returns an actual error of course.

The warning you are seeing, and in fact I'm seeing it too when using my test apps, is related to the newly introduced [review process for login apps](https://developers.facebook.com/docs/apps/review/login#post-f8-apps) In short your users won't see the warning message, but they won't see the the permissions that are not approved as well.

As for the [graph API versioning](https://developers.facebook.com/docs/apps/versions) it seems that the _unversioned_ call defaults to the oldest version of the API still supported, which after 30 April 2015 should be 2.0 (if I understand correctly)

In short I'm not against putting a version in that url, I just want to make sure it's needed :)

**Comment by simov** [comment]
@weswingcats I'm closing this PR, thanks for reporting this change, I'll try to keep an eye on it. Let me know if there is any issue :+1:

**Comment by simov** [comment]
@weswingcats I just published [v3.2.0](https://github.com/simov/grant/blob/master/CHANGELOG.md). With this version you can override any of the [reserved keys](https://github.com/simov/grant/blob/master/config/reserved.json) that Grant uses internally. That includes the ones defined in the [oauth.json](https://github.com/simov/grant/blob/master/config/oauth.json) file.

So in your configuration you can have something like this

```js
{
  "facebook": {
    "access_url": "https://graph.facebook.com/2.0/oauth/access_token",
    "scope": [...],
    ...
  }
}
```

Of course if you find a breaking change it's always better to report it here :) This change in the module is more about having the ability to change the oauth related options if you want.

**Comment by david-potgieter** [comment]
thanks @simov! To date I have not been able to replicate the deprecation warning, with or without the version specificity. I will keep tabs on it and let you know.

:+1:
